GeosCore/hco_interface_gc_mod.F90

```diff
@@ -4907,7 +4918,7 @@ SUBROUTINE Compute_Sflx_for_Vdiff( Input_Opt,  State_Chm, State_Diag,      &
        IF ( State_Diag%Archive_SatDiagnSurfFlux ) THEN
           DO S = 1, State_Diag%Map_SatDiagnSurfFlux%nSlots
              N = State_Diag%Map_SatDiagnSurfFlux%slot2id(S)
-             State_Diag%SatDiagnSurfFlux(:,:,S) = State_Chm%SurfaceFlux(:,:,N)
+             State_Diag%SatDiagnSurfFlux(I,J,S) = State_Chm%SurfaceFlux(I,J,N)
```

Comment:
Same comment as above. Note that both of these diagnostics have bugs that are now fixed.

Reply:
This should now be fixed in commit https://github.com/geoschem/geos-chem/commit/d9ee2e29b37f15b36326bdbe7dc1d2ee0eb2fc42.